packages/@aws-cdk/cx-api/package.json

```diff
@@ -87,7 +87,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@aws-cdk/cdk-build-tools": "0.0.0",
-    "@aws-cdk/cloud-assembly-schema": "^45.2.0",
+    "@aws-cdk/cloud-assembly-schema": "^48.2.0",
```

Comment:
Same comment as below, is this upgrade intentional? Seems like it may not be in scope of this PR.